manual/cypher/cypher-docs/src/docs/graphgists/intro/patterns-in-practice.adoc

```diff
@@ -141,8 +141,8 @@ Adding _Cloud Atlas_ as a new movie for _Tom Hanks_ could be achieved like this:
 
 [source,cypher]
 ----
-MATCH (p:Person {name:"Tom Hanks"})
-CREATE (m:Movie {title:"Cloud Atlas",released:2012})
+MATCH (p:Person {name: 'Tom Hanks'})
+CREATE (m:Movie {title: 'Cloud Atlas', released: '2012})
```

Comment:
`'2012` has a stray single quote which looks like it might be responsible for test failures